AK/Array.h

```diff
@@ -37,8 +37,8 @@ struct Array {
     [[nodiscard]] constexpr T const& front() const { return at(0); }
```

Comment:
Use an imperative Style for the commit messages:
`AK: Use correct max() function in Array` 
or the like

Similar for the previous commit message

Reply:
Fixed by squash? 😅